app/src/main/java/space/taran/arknavigator/ui/fragments/ResourcesFragment.kt

```diff
@@ -76,6 +78,11 @@ class ResourcesFragment(val root: Path?, val path: Path?): MvpAppCompatFragment(
         savedInstanceState: Bundle?): View? {
 
         Log.d(RESOURCES_SCREEN, "inflating layout for ResourcesFragment")
+        sharedPrefs = requireContext().getSharedPreferences("user_preferences", Context.MODE_PRIVATE)
+
+        sorting = Sorting.values()[sharedPrefs.getInt("sorting_preference", 0)]
```

Comment:
Separate variable for preference would make code more readable.

Reply:
Yes, it would be better to create constants for preference names.